test/2fa.test.js

```diff
@@ -92,9 +101,6 @@ describe('2fa method management', function () {
     });
 
     describe('changing an already requested 2fa method', () => {
```

Comment:
Does this test do anything but change the `kind` of the recovery method? If it's no longer providing any value now that we prevent changing `kind` then it's probably not worth keeping around.

Reply:
The important part here is the `should allow verification using the code generated by the second request` test inside. This will ensure that a user started the 2FA setup but didn't verify the sent code, can still start the process again and successfully verify with the new method (starting the process again will update the existing `RecoveryMethod`).

I think this is still relevant even though we are no longer updating the `kind`, but I think the `describe` message should be updated to something more relevant 🤔 maybe `completing an already requested 2fa method`?

Thoughts?

Reply:
Good point, I thought that was covered separately but I may be getting confused with an unmerged PR. In that case I think changing the name is the right way to go 👍